src/anm/ui/controls.js

```diff
@@ -242,7 +242,8 @@ Controls.prototype.react = function() {
                 time = p.anim.duration;
             }
             if (s === C.PLAYING) {
-              p.pause().play(time);
+              p.pause()
+               .play(time);
```

Comment:
this line-break was made just for easier debugging, providing an ability to set breakpoint to `.pause` :)